pkg/watcher/watcher.go

```diff
@@ -243,11 +347,59 @@ func (w *Watcher) deepCopyWatcherMetrics(src *WatcherMetrics) *WatcherMetrics {
 	}
 }
 
-// HTTP Handler for baseUrl endpoint
-func (w *Watcher) handler(resp http.ResponseWriter, r *http.Request) {
+// HTTP Handler for loadVariationRiskBalancing endpoint
```

Comment:
We should discuss if we use a generic handler with different params for all plugins or use different handlers.

Reply:
yes I do agree with you, but how to set and set for what